src/uct/ib/ud/base/ud_iface.c

```diff
@@ -660,7 +662,10 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
      ucs_offsetof(uct_ud_iface_config_t, ud_common),
      UCS_CONFIG_TYPE_TABLE(uct_ud_iface_common_config_table)},
 
-    {"TIMEOUT", "5.0m", "Transport timeout",
+    {"LINGER_TIMEOUT", "5.0m", "Timeout to wait before destroying endpoint",
```

Comment:
"Keep the connection open internally for this amount of time after closing it"

Reply:
done